flutter/lib/src/default_integrations.dart

```diff
@@ -397,6 +409,7 @@ class LoadReleaseIntegration extends Integration<SentryFlutterOptions> {
         .replaceAll('\t', '_')
         .replaceAll('\r\n', '_')
         .replaceAll('\r', '_')
-        .replaceAll('\n', '_');
+        .replaceAll('\n', '_')
+        .replaceAll('\u{0000}', '');
```

Comment:
we need to be sure that this works on Windows, maybe @bruno-garcia can test it?

Reply:
I dream of integration test on the target platform that will validate these for us. Mainly because of regressions introduced later.

Thoughts? Q2 is the time to plan and book these tasks